packages/reports/src/Constants/data-table-constants.js

```diff
@@ -11,6 +17,7 @@ import IndicativeCell from '../Components/indicative-cell.jsx';
 import MarketSymbolIconRow from '../Components/market-symbol-icon-row.jsx';
 import ProfitLossCell from '../Components/profit_loss_cell.jsx';
 import CurrencyWrapper from '../Components/currency-wrapper.jsx';
+import ContractCardSell from '@deriv/components/src/components/contract-card/contract-card-items/contract-card-sell.jsx';
```

Comment:
import { ContractCardSell } from '@deriv/components';

We should be importing like this